db/migrate/20220809145729_add_author_id_to_posts.rb

```diff
@@ -0,0 +1,6 @@
+class AddAuthorIdToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_column :posts, :AuthorId, :integer
```

Comment:
- [ ] adding a column like that won't achieve a relationship between the user and post  you should try something like that in order to  add association between the post and the user as `author` : 
```RUBY
t.references :author, foreign_key: { to_table: 'users' }
```

- [ ] please it would be better to follow the convention of creating columns in the database they should be in snake_case like that: comment_counter ...etc please try to do that